packages/x-date-pickers-pro/src/internals/models/dateTimeRange.ts

```diff
@@ -32,16 +30,8 @@ export interface UseDateTimeRangeFieldProps<
       'format'
     >,
     RangeFieldSeparatorProps,
-    DayRangeValidationProps<TDate>,
-    TimeValidationProps<TDate>,
-    BaseDateValidationProps<TDate>,
-    DateTimeValidationProps<TDate> {
-  /**
-   * 12h/24h view for hour selection clock.
-   * @default utils.is12HourCycleInCurrentLocale()
-   */
-  ampm?: boolean;
```

Comment:
This is not strictly related to validation but I took the opportunity to also remove duplication on this prop.